studio/components/layouts/StorageLayout/StorageLayout.tsx

```diff
@@ -42,6 +42,7 @@ const StorageLayout: FC<Props> = ({ title, children }) => {
   const { services, isLoading } = useProjectSettings(ref as string | undefined)
   const apiService = find(services ?? [], (service) => service.app.id === 1)
   const projectUrl = apiService?.app_config?.endpoint ?? ''
+  const urlProtocol = apiService?.app_config?.protocol
```

Comment:
This will be undefined in our production API. 

My preference is to retain the same API format for both self hosted and prod. With this PR, we add a field to only the self hosted API and not the production one which would lead to bugs only in one environment (like this one being undefined in production). Not opposed to adding a protocol field, but lets do it internally too.

Reply:
Yes we know it will be undefined in production, but also it's always bypassed in production.
It doesn't lead to bugs for now but it'll be in the future due to the mismatch of the API (for example someone modifies code without knowing the mismatch).
So for now it is safe.
Also if you add the protocol field it is safe to remove all conditionals, for example: `{IS PLATFORM ? "https" : protocol}`

Reply:
yeah that would be ideal @sweatybridge - adding it to the prod api, wdyt?

Reply:
Sounds good. I will update our api next week